src/java.desktop/share/classes/com/sun/imageio/plugins/wbmp/WBMPMetadata.java

```diff
@@ -25,19 +25,13 @@
 
```

Comment:
Since you updating a copyright year in all other files, you probably might want to update in this file too.

Reply:
I did not update it since this is the only file where the imports are updated only(the UnsupportedEncodingException is removed).